src/vs/workbench/browser/parts/editor/editorGroupView.ts

```diff
@@ -804,6 +807,13 @@ export class EditorGroupView extends Themable implements IEditorGroupView {
 		return this.model.getEditors(order, options);
 	}
 
+	findEditors(resource: URI, order: EditorsOrder, options?: { excludeSticky?: boolean }): EditorInput[] {
+		const canonicalResource = this.uriIdentityService.asCanonicalUri(resource);
+		return this.getEditors(order, options).filter(editor => {
+			return editor.resource && isEqual(editor.resource, canonicalResource);
```

Comment:
There is a `excludeSticky` option not being used here, do we actually need it?

Reply:
It gets passed into the `getEditors` function which then gets passed into the model which cares